src/input_map.rs

```diff
@@ -429,57 +348,18 @@ impl<A: Actionlike> InputMap<A> {
 
 // Utilities
 impl<A: Actionlike> InputMap<A> {
-    /// Returns the mapping between the `action` that uses the supplied `input_mode`
-    ///
-    /// If `input_mode` is `None`, all inputs will be returned regardless of input mode.
-    ///
-    /// For chords, an input will be returned if any of the contained buttons use that input mode.
-    ///
-    /// If no matching bindings are found, an empty [`PetitSet`] will be returned.
-    ///
-    /// A copy of the values are returned, rather than a reference to them.
-    /// The order of these values is stable, in a first-in, first-out fashion.
-    /// Use `self.map.get` or `self.map.get_mut` if you require a reference.
-    #[must_use]
-    pub fn get(&self, action: A, input_mode: Option<InputMode>) -> PetitSet<UserInput, 16> {
-        let full_set = self.map[action.index()].clone();
-        if let Some(input_mode) = input_mode {
-            let mut matching_set = PetitSet::default();
-            for input in full_set.iter() {
-                if input.matches_input_mode(input_mode) {
-                    matching_set.insert(input.clone());
-                }
-            }
-
-            if matching_set.is_empty() {
-                PetitSet::default()
-            } else {
-                matching_set
-            }
-        } else {
-            full_set
-        }
-    }
-
-    /// Returns how many bindings are currently registered for the provided action with the provided [`InputMode`]
-    ///
-    /// If `None` is provided, a total across all input modes will be provided.
-    ///
-    /// A maximum of `CAP` bindings across all input modes can be stored for each action,
-    /// and insert operations will silently fail if used when `CAP` bindings already exist.
+    /// Returns the `action` mappings
     #[must_use]
-    pub fn n_registered(&self, action: A, input_mode: Option<InputMode>) -> usize {
```

Comment:
This API is still useful, especially to avoid unexpected panics.

Reply:
But maybe just use `get(action).len()`?

Reply:
Yeah okay fair :)